homeassistant/components/asuswrt/config_flow.py

```diff
@@ -0,0 +1,238 @@
+"""Config flow to configure the AsusWrt integration."""
+import logging
+import os
+import socket
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components.device_tracker.const import (
+    CONF_CONSIDER_HOME,
+    DEFAULT_CONSIDER_HOME,
+)
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_MODE,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_PROTOCOL,
+    CONF_USERNAME,
+)
+from homeassistant.core import callback
+from homeassistant.helpers import config_validation as cv
+
+# pylint:disable=unused-import
+from .const import (
+    CONF_DNSMASQ,
+    CONF_INTERFACE,
+    CONF_REQUIRE_IP,
+    CONF_SSH_KEY,
+    CONF_TRACK_UNKNOWN,
+    DEFAULT_DNSMASQ,
+    DEFAULT_INTERFACE,
+    DEFAULT_SSH_PORT,
+    DEFAULT_TRACK_UNKNOWN,
+    DOMAIN,
+    MODE_AP,
+    MODE_ROUTER,
+    PROTOCOL_SSH,
+    PROTOCOL_TELNET,
+)
```

Comment:
```suggestion
    MODE_AP,
    MODE_ROUTER,
    PROTOCOL_SSH,
    PROTOCOL_TELNET,
)
from .const import DOMAIN  # pylint:disable=unused-import
```
Perhaps you should only disable pylint for the constant it applies to.

Reply:
The 3 PR inside this one was already reviewed and approved, really I need to change something?

Reply:
No. Just my suggestion.

Reply:
Let's address this in a future PR.